docs/src/metadata.md

```diff
@@ -1,155 +1,80 @@
 # Metadata
 
+```@setup metadata-example
+using JLSO, Dates
+JLSO.save("breakfast.jlso", :food => "☕️🥓🍳", :cost => 11.95, :time => Time(9, 0))
+```
```

Comment:
I personally think it would still be worth showing this in the doc (I think @setup hides it, correct?)

Reply:
Yes, but it's already in the README.md/index.md.